src/server/services/UpdateReserveServices.js

```diff
@@ -1,4 +1,4 @@
-class UpdateVideoResServices {
+class UpdateReserveServices {
```

Comment:
@maxpluez I see now both Video Reserves and Music Reserves is sharing the same code.. can this also work for BruinCast?